libs/azure-ai/langchain_azure_ai/vectorstores/azure_cosmos_db_no_sql.py

```diff
@@ -668,12 +656,12 @@ def _construct_query(
             full_text_rank_filter,
             with_embedding,
         )
-        query += " FROM c "
+        table = self._table_alias
```

Comment:
We can leave this hardcoded as c, as this is internal to the implementation of the vector store, and won't matter to the user what table-alias name is being used.

Reply:
Same as above